### ISSUE TITLE
Contribution guide to its own repo

### ISSUE THREAD

**Issue by lkngtn** [1Hive/1Hive#37]
Using the doc sync feature of docusaurus, does it make sense to isolate the contributor guide into its own repository?

**Comment by yeqbfgxjiq** [comment]
Yeah that could be good. Should we maybe rename the "communication" repo to "community-building", and then put everything in there that has to do with getting people involved with the project?

**Comment by yeqbfgxjiq** [comment]
Atm since we have a lot going on do you want to just keep it simple and leave contributor stuff in the website repo?

**Comment by lkngtn** [comment]
Yeah I wouldn't  consider moving this out of the main website a priority. Lets leave this open but just not prioritize it right now.
